server/routes/index.ts

```diff
@@ -24,6 +27,14 @@ export default function routes(router: Router, services: Services): Router {
           roles: [MAINTAINER_ROLE],
           enabled: true,
         },
+        {
+          id: 'prison-register',
+          heading: 'Prison register',
+          description: 'View prison details.',
+          href: '/prison-register',
+          roles: [MAINTAINER_ROLE],
```

Comment:
Using the same role as Court Register tile as haven't heard it should be a different one